marlowe-playground-client/test/Marlowe/ContractTests.purs

```diff
@@ -74,9 +79,26 @@ import Test.QuickCheck.Gen (Gen)
 import Test.Spec (Spec, describe, it)
 import Test.Spec.Assertions (shouldEqual)
 import Test.Spec.QuickCheck (quickCheck)
-import Text.Pretty (pretty)
 import Type.Prelude (Proxy(..))
 
+unixEpoch :: Instant
+unixEpoch = unsafeInstantFromInt 0
+
+shelleyEpoch :: Instant
+shelleyEpoch =
+  let
+    -- 2020-07-29 21:44:51 UTC expressed as unix epoch
+    epoch = Milliseconds 1596059091000.0
+  in
+    unsafePartial $ fromJust $ instant $ epoch
+
+secondsSinceShelley :: Int -> Instant
+secondsSinceShelley seconds = fromDateTime
+  $ fromMaybe (if seconds < 0 then bottom else top)
+  $ adjust (Seconds $ Int.toNumber seconds)
+  $ toDateTime
+  $ shelleyEpoch
```

Comment:
This code has always been problematic - the Shelley epoch isn't a static constant; it depends on the network and is defined in the Genesis config. I'm curious what we are using it for in the playground though - as we do not interact with a Node at all. How come we need to work with times relative to this Shelley epoch?

If it's only used in this test file, I'd recommend we replace all instances of `secondsSinceShelley` with `unsafeInstantFromInt`, and simply shift our reference to the UNIX epoch instead. Then we can get rid of these inaccurate definitions for good.